portfolio/src/main/webapp/index.html

```diff
@@ -111,17 +120,45 @@ <h6 class="card-subtitle text-muted">Technologies Used: C++, Arduino </h6>
           <div class="card mb-4 shadow-sm">
              <img class="ctr" style="text-align:center;height:225x; width:325px;" src="img/image.png"/>
             <div class="card-body">
-              <p class="card-text">Cork. Project for HackedBeta 2019 Hackathon. Made a web app to compile campus posters into a single website where students can filter and sign up for events.</p>
+              <p class="card-text">Cork. Project for HackedBeta 2019 Hackathon. Made a web app to compile campus posters
+                   into a single website where students can filter and sign up for events.</p>
               <h6 class="card-subtitle text-muted">Technologies Used: Django (Python), HTML/CSS, JS</h6>
               <div class="d-flex justify-content-between align-items-center">
-                <a href="https://github.com/katielin42/Django-Web-App-CorkBoard" <button type="button" class="btn btn-sm btn-outline-secondary">Link</button></a>
+                <a href="https://github.com/katielin42/Django-Web-App-CorkBoard" <button type="button" class="btn btn-sm btn-outline-secondary">
+                    Link</button></a>
                 <small class="text-muted">Nov 2019</small>
               </div>
             </div>
           </div>
         </div>
     
     </div>
+    <br/>
+    <div id="blog">
+    
+        <div id="content">
+
+             <h2 class="center" style="color:darkslategrey;
+    font-family:Questrial;
+    font-size:60px;">Blog</h2>
+       <p class="center"><i>What I am currently working on...</i></p>
+       </div>
+            <br/>
+
+        <div class="blogpost" id="blog0">
+        <h3>Coursera Machine Learning Course</h3>
+        <p class="blogtext">I am currently on Week 6 of Andrew Ng's famous introductory course on machine learning. This week is about 
+            getting proper diagnostics and how to best change parameters to get the best possible fit in your models.  </p>
+        <img src="img/bot.png" class="blogimg"/>
+       
+        </div>
+         <a href="blogs/blog0.html" <button type="button" class="btn btn-sm btn-outline-secondary">Read More</button></a>
+    </div>
+    
+
+   
+    <div class="blankspace"></div> <!-- necessary to allow for scrolling to blog section"
+
     </body>
   
   <script src="https://code.jquery.com/jquery-3.1.1.min.js"></script>
```

Comment:
If you uncomment this: I believe you're not supposed to put anything outside of the head/body tags within `<html>`. Browsers might accept it and try to handle it anyway, but likely by assuming you meant to put it in the `<body>`.

Reply:
I'm unclear what you mean by this. This div block is inside <body>.

Reply:
The scripts are outside the body.